mesheryctl/internal/cli/root/system/start.go

```diff
@@ -48,8 +51,303 @@ import (
 var (
 	skipUpdateFlag  bool
 	skipBrowserFlag bool
+	configureCluster string
 )
 
+func getContexts(configFile string) ([]string, error) {
+	client := &http.Client{}
+
+	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+	if err != nil {
+		utils.Log.Error(err)
+		return nil, nil
+	}
+
+	// GETCONTEXTS endpoint points to the URL return the contexts available
+	GETCONTEXTS := mctlCfg.GetBaseMesheryURL() + "/api/system/kubernetes/contexts"
+
+	req, err := utils.UploadFileWithParams(GETCONTEXTS, nil, utils.ParamName, configFile)
+	if err != nil {
+		return nil, ErrUploadFileParams(err)
+	}
+
+	res, err := client.Do(req)
```

Comment:
```suggestion
	res, err := utils.MakeRequest(req)
```
We have this method that is used every in the code so we should reuse it instaed of create a http.Client

Reply:
Can you replace client.Do by utils.MakeRequest